arm-resources/resources/2016-02-01/swagger/resources.json

```diff
@@ -2427,9 +2427,6 @@
           "description": "Resource tags"
         }
       },
-      "required": [
```

Comment:
This is a breaking change in multiple ways.
1. Making a required property optional.
2. Modifying this in an older spec, which is being already consumed by customers